src/app/pages/account/index/index.html

```diff
@@ -34,6 +34,16 @@
         </tr>
         </tbody>
       </table>
+
+      <router-link
+            :to="{ name: 'account.update' }"
+            active-class="active"
+            class="btn btn-link"
+            tag="a"
+          >
+            Account update
```

Comment:
Place `Account update` in a `<a>`-tag and remove the `tag`-attribute from `<router-link>`
```HTML
<router-link
   :to="{ name: 'account.update' }"
   active-class="active"
   class="btn btn-link"
>
   Update account
</router-link>
```